cni/network/network.go

```diff
@@ -754,8 +751,7 @@ func (plugin *NetPlugin) createEndpointInternal(opt *createEndpointInternalOpt)
 	}
 
 	// Create the endpoint.
-	telemetry.SendCNIEvent(plugin.tb, fmt.Sprintf("[cni-net] Creating endpoint %+v.", epInfo))
-	log.Printf("[cni-net] Creating endpoint %v.", epInfo.Id)
+	telemetry.LogAndSendEvent(plugin.tb, fmt.Sprintf("[cni-net] Creating endpoint %+v.", epInfo.String()))
```

Comment:
minor, shouldn't this be %s since we are getting epInfo.String()

Reply:
yeah, %v is "make a string representation" so it works, but it should probably just be %s